plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/StandardSQLDialectQueryGenerator.java

```diff
@@ -103,6 +106,38 @@ public void appendConditionString(
         appendConditionString(filter, constraints, dataSource, conditionTable, query, inlineCriteria, subQuery);
     }
 
+    @Override
+    public String addFiltersToQuery(
+        @Nullable DBRProgressMonitor monitor,
+        final @NotNull DBPDataSource dataSource,
+        @NotNull String sqlQuery,
+        final @NotNull DBDDataFilter dataFilter
+    ) {
+        return dataSource.getSQLDialect().getQueryGenerator().addFiltersToQuery(monitor, dataSource, sqlQuery, dataFilter);
+    }
+
+
+    @Override
+    public String wrapQuery(@NotNull DBPDataSource dataSource, @NotNull String sqlQuery,
+                                   @NotNull DBDDataFilter dataFilter) {
```

Comment:
I'm not sure about the reason to wrap one argument to **the middle** of the next line, but I'd format it this way.
```suggestion
    public String wrapQuery(
        @NotNull DBPDataSource dataSource,
        @NotNull String sqlQuery,
        @NotNull DBDDataFilter dataFilter
    ) {
```